portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsCreation.testcase

```diff
@@ -574,21 +592,21 @@ definition {
 
 		task ("View the segment criteria card") {
 			ACSegments.viewSegmentCriteriaIndividualAttributes(
-				attributeName = "country",
-				attributeValue = '''"USA"''',
+				attributeName = "jobTitle",
+				attributeValue = '''"QA"''',
 				condition = "is not",
 				conjunction = "AND");
 
 			ACSegments.viewSegmentCriteriaIndividualAttributes(
-				attributeName = "country",
+				attributeName = "jobTitle",
 				condition = "is known",
 				index = "2");
 		}
 
 		task ("Switch tab to membership and check the membership list") {
 			ACNavigation.switchTab(tabName = "Membership");
 
-			ACUtils.viewNameList(nameList = "${membershipList}");
+			ACUtils.viewNameList(nameList = "user3 user3");
 		}
```

Comment:
Hey @manuferreira, I think at the end of the test you could put it to verify that user1 and user2 are not in the list, that way we can verify that the criteria are working in the segment:

`for (var username : list "user1 user1,user2 user2") {
         ACUtils.viewNameListIsNotPresent(nameList = "${username}");
}`

Reply:
I think inside the ACUtils.viewNameListIsNotPresent we already have a for loop.